### ISSUE TITLE
[Issue]: Can't make any changes to the Admin->Networking page, blue save button is not functional

### ISSUE THREAD

**Issue by ioogithub** [jellyfin/jellyfin-web#3440]
### Please describe your bug

The blue "save button" on the Networking page is broken. This is a brand new installation. If I try to make any changes to the network settings and click the blue "save" button at the bottom of the page, nothing happens. There is no notification of the setting being saved. If  I navigate away from the page all changes are lost.

I also tried the following:

1. Opening the page is a Firefox private window and loading the page.
2. Generating a new Firefox profile with default settings and loading the page.
3. Accessing the page thought https://domain.com/web/index.html#!/networking.html
4. Accessing the page though the http://localhost:8096/web/index.html#!/networking.html

The blue save button does not respond when a change is made and then clicked. There is no way to save changes to the networking page.

Note: The save button on other pages works as expected. I can click on DLNA, make a change, click save and it saves as expected. The network save button however does not work.

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS:Raspberry Pi (Debian bullseye)
- Virtualization: None
- Clients: Browser
- Browser: Firefox 98.0
- FFmpeg Version: n/a
- Playback Method: n/a
- Hardware Acceleration: n/a
- Plugins: n/a
- Reverse Proxy: apache
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
There is no event in the log of the attempted configuration change, the save button is broken.
```

### FFmpeg logs

```shell
n/a
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dmitrylyzo** [comment]
@ioogithub Do you have a dialog when pressing `Save`?
![network-1](https://user-images.githubusercontent.com/56478732/154932072-3b8446fd-6f8b-424b-afc7-d2a6682930b4.png)

**Comment by ioogithub** [comment]
No I do not see this dialog. When I press save, nothing happens, it just seems like the button is broken.

**Comment by JPVenson** [comment]
I had the same problem when changing an option that requires a restart. In my case that was proxy setting. Could you look into the browsers dev console? I guess there should been an error

**Comment by blastrock** [comment]
There is (on chrome only), you can look at the two mentioned issues in this thread to see how this issue can happen.

**Comment by rolex905** [comment]
Hello, new Jellyfin user here who is experiencing the same problem on a brand new RPi4 install.  Why is the SAVE button non-functional and why does it give zero output?

**Comment by rolex905** [comment]
> No I do not see this dialog. When I press save, nothing happens, it just seems like the button is broken.

same here.  clicking it does nothing, at all.  the cursor changes to hand when you hover over it which implies it should be clickable -- but nothing happens.

**Comment by ioogithub** [comment]
Is there a solution to this issue?

I haven't been able to get past it honestly. Because it happens so early in the setup I wasn't actually able to use Jellyfin and I don't know a way around it and I started looking at alternatives.

**Comment by dmitrylyzo** [comment]
Fields `UDP Communication Range` and `SSDP Filter` have `undefined` values that doesn't match their masks, and the browser validates them, despite they aren't required. They are hidden - that's why "is not focusable" error occurs.
Emptying them or trimming the values according to the masks fixes the form submitting.

**Comment by gardner** [comment]
Steps to reproduce:

- Navigate to Settings > Advanced > Networking
- Under "Remote Access Settings" ensure "Allow remote connections to this server." is checked ✅ 
- Under "Public HTTP port number" delete the port number value leaving an empty input box
- Uncheck "Allow remote connections to this server."
- Attempt to submit the form
- Please note a console log error message is logged "An invalid form control with name='' is not focusable."

This is caused by the form hiding elements which are marked to be validated during the form submit.

**Recommended fix:** Remove this [event handler](https://github.com/jellyfin/jellyfin-web/blob/41d2b104d52168c880da559df1e471bae7e8ece6/src/controllers/dashboard/networking.js#L168) from the code. The section of the form will no longer be hidden when the user checks "Allow remote connections to this server." and form validation will work as expected.

**Comment by thornbill** [comment]
This should have been fixed in master by #4703
